nixos/modules/services/web-apps/isso.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption types literalExample;
+
+  cfg = config.services.isso;
+
+  settingsFormat = pkgs.formats.ini { };
+  configFile = settingsFormat.generate "isso.conf" cfg.settings;
+in {
+
+  options = {
+    services.isso = {
+      enable = mkEnableOption ''
```

Comment:
The "Whether to enable" text and trailing period are redundant with `mkEnableOption`. Take a quick peak at the definition of needed.

Reply:
I'm a little bit unsure what's the suggestion is? Should I remove the additional documentation or merge it with the description set via `mkEnableOption`?

Reply:
Defintely keep the documentation, just remove the first few words and the final period.